### DISCUSSION TITLE
Compare only file names

### DISCUSSION THREAD

**Discussion by josejjpc** [WinMerge/winmerge#1217] [General]
I have files inside of 2 folders to compare and I would like to compare only file names, is that posible?

In the option menu I see some compare method and do not find option to file name only. is there any way to do?
![image](https://user-images.githubusercontent.com/100685735/156155609-69999685-e676-4da8-88ab-7a789e56a092.png)

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have a comparison method that compares only filenames.

**Reply by Steev43230** [reply]
Is that on the horizon? It would be super helpful (to me) to see only missing or new folders/files.

**Reply by sdottaka** [reply]
As of now, there are no plans for implementation.
By the way, if you only want to search for files that do not exist on one side, you can change the Compare Method to "Modified Date and Size" or other options that do not compare the contents of the files, and then uncheck the "Show Identical Items" and "Show Different Items" menu items as follows.
![image](https://user-images.githubusercontent.com/98126/232050538-c5ebc4c4-77c2-435b-a444-28aea18f07a6.png)

**Reply by hueldoeu** [reply]
> As of now, there are no plans for implementation. By the way, if you only want to search for files that do not exist on one side, you can change the Compare Method to "Modified Date and Size" or other options that do not compare the contents of the files, and then uncheck the "Show Identical Items" and "Show Different Items" menu items as follows. ![image](https://user-images.githubusercontent.com/98126/232050538-c5ebc4c4-77c2-435b-a444-28aea18f07a6.png)

yes. thank you. this is very good.
<img width="815" height="567" alt="image" src="https://github.com/user-attachments/assets/fea912a7-ca81-474c-a160-bf652cbc3217" />
i am uploading files on a server and it fails sometimes so i have to know which files had been transmitted already.

**Comment by Soundchazer2k** [comment]
That would be a good feature. Sometimes I like comparing files that I have zipped in one location and unzipped in another, just to make sure I have exactly the same number of files in each folder.

**Comment by peteringemann** [comment]
That feature would be nice.
I have just finished robocopying several millions of small files (took me two weeks) and now I would like to compare to see if the folder and file name structure is in place. I don't wanna compare the file content - just the file name (if I wanted a binary comparison it would probably be the same as copying them again). 
So to speed up this process it would be great to just compare folder structure and file name.

**Reply by SamSirry** [reply]
Doesn't robocopy already do this kind of comparison if you use the proper options to sync one side to another?

**Reply by hueldoeu** [reply]
> Doesn't robocopy already do this kind of comparison if you use the proper options to sync one side to another?

i need to compare a folder in C:\ and a webDAV drive. i can't robocopy files from C:\ to the network drive WebDAV, this would crash my system.
